.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Decided to go with node 14 instead of node 10 because not everyone can install node 10. The CI will still check if everything works on Node 10.